docs/tools/index.md

```diff
@@ -4,7 +4,7 @@ title: fgsv tools
 
 # fgsv tools
 
-The following tools are available in fgsv version 0.2.0-d603e95.
+The following tools are available in fgsv version 0.2.0-5ce8bc6.
```

Comment:
Could you please bump the major version, as excluding duplicates (and qc fails) by default willl be a breaking change?